src/components/SocialMedia/social.css

```diff
@@ -0,0 +1,31 @@
+.social {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
```

Comment:
Some css declarations here (Lines 2, 3, 5) are already specified on the component as tailwind css.
Kindly resolve in order to reduce the lines of code in this custom css.